### ISSUE TITLE
Grey map or not even drawing on Android

### ISSUE THREAD

**Issue by mbrucher** [fqborges/react-native-maps#9]
### Is this a bug report?

Yes

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

As far as I know, everything was installed properly.

### Environment

    react-native: 0.60.5
    react: 16.8.6
	react-native-maps: 0.23.0rc4

  Target Platform: Android Pixel 3a, API 29 (emulator and Nokia 7+)

### Steps to Reproduce

I added a Map in a View like this:

```js
    return <MapView
      provider={Platform.OS === "ios" ? null : "osmdroid"}
      initialRegion={region}
    >
      <UrlTile
        maximumZ={18}
        urlTemplate="https://{s}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}.png"
      />
    </MapView>;
```

### Expected Behavior

Should display a map.

### Actual Behavior

The map stays grey (interaction works).

I suppose I missed a step while installing the package, but I can't figure out what.

**Comment by fqborges** [comment]
Do you still experience this problem?

**Comment by mbrucher** [comment]
Yes, currently, I have to use the original react-native-maps instead :(

**Comment by slavikdenis** [comment]
Having the same issue, but only on release version of the app.

**Comment by franamu** [comment]
Try giving it a style

const { width, height } = Dimensions.get('window');

inside your return put

<MapView
          initialRegion={{
            latitude: 37.78825,
            longitude: -122.4324,
            latitudeDelta: 0.0922,
            longitudeDelta: 0.0421,
          }}
          style={{height: height, width: width}}
        />

**Comment by aardvarkk** [comment]
Not having a defined size was my problem. This should be called out in the README I would think. Spent a long time trying to figure out why nothing was rendering!
